### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by albertmiro**
Update README to explain better the LogIn / Register point.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,9 @@ Create a project, install dependencies and run a web server using your favourite
 - API RESTful. Generate routes and use cases to get the list of available postures, login and register.
 - Deploy in the Cloud:
     - BD (i.e: GoogleCloud, Mongo Atlas, IBM Cloud, Firebase)
-    - API (i.e: Heroku)
+    - API (i.e: Heroku or antoher alternative)
+    
+Due that this is not a real case, we don't need a complex Registration / Log In system, for the scope of this challenge we are happy with a register / login system that stores the data directly to the database and just compares the username and password to check if the user is registered and can go to their private area.
```
Comment (WildStudio) [inline README.md]: Looks good to me, may be just a little change: replace Log In with login as it is used as a noun, [see](https://www.grammar.com/log_in_vs._login)

Comment (albertmiro) [reply README.md]: Done!

Comment (WildStudio) [reply README.md]: Not a big deal tho!

Comment (albertmiro) [reply README.md]: It was a really difficult change!

<image src="https://user-images.githubusercontent.com/15966505/71250271-90e35f80-231f-11ea-9a2b-55b4124cbe91.png" width=30%/>

**Pull request merged**